internal/config/configuration.go

```diff
@@ -68,12 +69,44 @@ func Validate(config *Configuration) error {
 			return fmt.Errorf("entrypointUrl %s is not unique", spec.EntrypointURL)
 		}
 		uniquenessUrl[spec.EntrypointURL] = true
+
+		// Validate the storage
+		if err := registerAndvalidateStorage(spec); err != nil {
+			return fmt.Errorf("storage %s is not valid: %s", spec.Name, err.Error())
+		}
 	}
 
 	log.Info().Msgf("Load %d configurations", len(config.Specs))
 	return nil
 }
 
+// registerAndvalidateStorage registers the storage and validate it
+// if the storage is not found or an error is occured during the
+// initialization or connection, the error is returned during the
+// validation
+func registerAndvalidateStorage(spec *WebhookSpec) error {
```

Comment:
```suggestion
func registerAndvalidateStorage(spec *WebhookSpec) (err error) {

```
Define err on func definition instead of var